.github/workflows/deploy-userguide.yml

```diff
@@ -0,0 +1,59 @@
+# When a new Release is created, deploy relevant
+# files to each of the generated repos.
+name: Deploy User Guide
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  framework:
+    name: Deploy to userguide
+    if: (github.repository == 'codeigniter4/CodeIgniter4')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Identify
+        run: |
+          git config --global user.email "action@github.com"
+          git config --global user.name "${GITHUB_ACTOR}"
+
+      - name: Checkout source
+        uses: actions/checkout@v3
+        with:
+          path: source
+
+      - name: Checkout target
+        uses: actions/checkout@v3
+        with:
+          repository: codeigniter4/userguide
+          token: ${{ secrets.ACCESS_TOKEN }}
+          path: userguide
+
+      - name: Install Sphinx
+        run: |
+          sudo apt install python3-sphinx
+          sudo pip3 install sphinxcontrib-phpdomain
+          sudo pip3 install sphinx_rtd_theme
```

Comment:
Why not use the `requirements.txt`?

Reply:
I just followed the guide we ahve in **admin/RELEASE.md** - I am clueless when it comes to Sphinx. If someone knowledgeable wants to make this better then please do :)

Reply:
This was based on the [README](https://github.com/codeigniter4/CodeIgniter4/blob/develop/user_guide_src/README.rst)
```
pip install -r user_guide_src/requirements.txt

pip3 install -r user_guide_src/requirements.txt

# For Windows using the Python Launcher
py -3 -m pip install -r user_guide_src/requirements.txt
```

Anyway, I don't see current makes much of a difference with the README's instructions, so I'm good here.